### PR TITLE
[C+B] Created FusedExplosive interface. Adds BUKKIT-3962, BUKKIT-3905

### PR DESCRIPTION
**Pull request by nitnelave**
There is currently no unified way to represent explosives with a fuse, and to get/set the fuse ticks.

The problem is addressed here by creating an interface just for that : FusedExplosive.

The interface FusedExplosive extends Explosive, and is used for all fused entities (TNTPrimed, TNTMinecart and Creeper). The implementation is given in the corresponding CB PR.

CB-1117 - https://github.com/Bukkit/CraftBukkit/pull/1117 - Implementation.

BUKKIT-3962 - https://bukkit.atlassian.net/browse/BUKKIT-3962
BUKKIT-3905 - https://bukkit.atlassian.net/browse/BUKKIT-3905

### CODE REVIEW

File: src/main/java/org/bukkit/entity/FusedExplosive.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package org.bukkit.entity;
+
+/**
+ * A representation of a fused {@link Explosive}.
+ */
+public interface FusedExplosive extends Explosive {
```
Comment (turt2live) [inline src/main/java/org/bukkit/entity/FusedExplosive.java:6]: Should this also handle incendiary explosive properties, such as what [this](https://github.com/Bukkit/CraftBukkit/pull/1117/files#L0R15) appears to do? (links to your CB PR)

**Comment by turt2live** [comment]
@nitnelave If you are still interested in this please update it to the latest code base and resolve the outlined issues.

The PR description would also need updating as per the CONTRIBUTING.md guidelines. A lengthy justification may also be needed.

As well, the PR should only reference one ticket.

**Comment by ST-DDT** [comment]
The updated version is now here:
https://github.com/Bukkit/Bukkit/pull/993

**Comment by turt2live** [comment]
Other half closed.

**Pull request closed without merging**